skrub/_joiner.py

```diff
@@ -21,21 +22,22 @@ class FeatureAugmenter(TransformerMixin, BaseEstimator):
 
     1. The main table and the key column name are provided at initialisation.
     2. The auxiliary tables are provided for fitting, and will be joined
-       sequentially when FeatureAugmenter.transform is called.
+       sequentially when Joiner.transform is called.
 
     It is advised to use hyperparameter tuning tools such as GridSearchCV
     to determine the best `match_score` parameter, as this can significantly
     improve your results.
-    (see example 'Fuzzy joining dirty tables with the FeatureAugmenter'
+    (see example 'Fuzzy joining dirty tables with the Joiner'
     for an illustration)
 
     Parameters
     ----------
-    tables : list of 2-tuples of (:obj:`~pandas.DataFrame`, str)
+    tables : list or list of 2-tuples of (:obj:`~pandas.DataFrame`, str)
```

Comment:
As discussed, instead of passing a list of two items when we have a single table, which is a bit odd, we should allow passing a 2-tuple instead, so
```py
table : 2-tuple of (pd.DataFrame, str) or list of 2-tuple (pd.DataFrame, str)
```